v2/spanner-to-sourcedb/src/main/java/com/google/cloud/teleport/v2/templates/utils/ShadowTableCreator.java

```diff
@@ -47,6 +47,8 @@ public class ShadowTableCreator {
   private final SpannerConfig spannerConfig;
   private final SpannerConfig metadataConfig;
   private String shadowTablePrefix;
+  private Ddl informationSchemaOfMainDb;
```

Comment:
nit: mainDb sounds a bit odd , will it be better to use primaryDb?

Reply:
Done